From c2af4e93ef24a63fe1c49caf1f4ad96afd92ac2c Mon Sep 17 00:00:00 2001
From: Pankit Garg <pankit.garg@nxp.com>
Date: Thu, 17 Sep 2020 17:26:49 +0530
Subject: [PATCH 17/22] dts: ls1046awrdb: Temporarily disable ethernet mac9
 node

Temporarily disabling ethernet mac9 node as XFI at MAC9
is not working currently and a crash is observed during
Linux-boot with MAC9 enabled.
MAC-9 will be re-enabled back once crash is resolved.

Signed-off-by: Pankit Garg <pankit.garg@nxp.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1046a-wrdb-sdk.dts | 3 +++
 arch/arm64/boot/dts/freescale/fsl-ls1046a-wrdb.dts     | 1 +
 2 files changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1046a-wrdb-sdk.dts b/arch/arm64/boot/dts/freescale/fsl-ls1046a-wrdb-sdk.dts
index c5be4d188237..bd434f04dccb 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1046a-wrdb-sdk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1046a-wrdb-sdk.dts
@@ -54,6 +54,9 @@
 	ethernet@3 {
 		status = "disabled";
 	};
+	ethernet@8 {
+		status = "disabled";
+	};
 	ethernet@9 {
 		compatible = "fsl,dpa-ethernet";
 		fsl,fman-mac = <&enet7>;
diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1046a-wrdb.dts b/arch/arm64/boot/dts/freescale/fsl-ls1046a-wrdb.dts
index a3047322eef6..a0176c8d3070 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1046a-wrdb.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1046a-wrdb.dts
@@ -141,6 +141,7 @@
 	ethernet@f0000 { /* 10GEC1 */
 		fixed-link = <0 1 1000 0 0>;
 		phy-connection-type = "xgmii";
+		status = "disabled";
 	};
 
 	ethernet@f2000 {
-- 
2.17.1

